IFC4x3/Properties/f/FaultDeformationStyle_1lYGGx_1jCCxEUZce$aPdE/Documentation.md

```diff
@@ -0,0 +1 @@
+Contain a human readable description of the style of deformation (e.g. brittle, ductile etc.) for the ShearDisplacementStructure.  To report an identifier from a controlled vocabulary, deformationStyle_uri shall be used.
```

Comment:
What is ShearDisplacementStructure?

Also "deformationStyle_uri" probably requires a reference.
```suggestion
Contain a human readable description of the style of deformation (e.g. brittle, ductile etc.) for the ShearDisplacementStructure. To report an identifier from a controlled vocabulary, deformationStyle_uri shall be used.
```

Reply:
> What is ShearDisplacementStructure?
> 
> Also "deformationStyle_uri" probably requires a reference.

Can't we change that when the IfcURIReference PR has been merged?

Reply:
I don't know what "IfcURIReference PR" is. But happy to do that if you know what is required.

Reply:
See reference from OneGeology.org on predefined properties and GeoSCIML Schema 4.0 at : https://onegeology.org/archive/wmsCookbook/7_3_2.html